Modix.Data/Repositories/MessageRepository.cs

```diff
@@ -213,9 +213,11 @@ string GetQuery()
                             count(*) as ""MessageCount"",
                             ""AuthorId"" = :UserId as ""IsCurrentUser""
                         from ""Messages""
+                        left outer join ""DesignatedChannelMappings"" on ""Messages"".""ChannelId"" = ""DesignatedChannelMappings"".""ChannelId""
                         where ""GuildId"" = :GuildId
```

Comment:
Looks like this'll be an ambiguous reference. Needs to be `""Messages"".""GuildId"" = :GuildId` I think

Reply:
Good catch! I missed that one when I was copying my changes into VS.

Reply:
I pushed an update.